test/mocks/server/mocks.cc

```diff
@@ -8,7 +8,11 @@ using testing::SaveArg;
 
 namespace Server {
 
-MockOptions::MockOptions() {}
+MockOptions::MockOptions() : path_("test/config/integration/server.json") {
```

Comment:
You should probably set this inside the test, since different tests may want different path_ values.